devel/patchelf/Portfile

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+name                patchelf
+version             0.10.0
+categories          devel
+platforms           darwin
+license             GPL-3
+maintainers         @lkollar
+description         Modify dynamic ELF executables
+long_description    PatchELF is a simple utility for modifying existing ELF executables and libraries
+homepage            https://github.com/NixOS/patchelf
+
+github.setup        NixOS patchelf 0.10
```

Comment:
please move the `github.setup` line directly after the newline below `PortGroup`, the `github` PG sets quite a few things automatically, so for example you can remove the `name`, `version`, and `homepage` fields from the Portfile as they are the defaults.